README.md

```diff
@@ -14,3 +15,20 @@ For Northeastern University
 ![alt text](res/Figma2.PNG)
 
 [Figma Link](https://www.figma.com/file/YsBjuMR3hcsKRLJFGfePu4/Data-Kernel_?type=design&node-id=4%3A63&mode=design&t=KhngNZH3w0eEhSVj-1)
+
```

Comment:
A bit unrelated to the issue for this PR but it would be nice to have the node version needed to run the project shown somewhere. It could be in the read.me or a .nvmrc file. Some of the packages that are installed need later versions of node so that would make the whole process easier! :)

Reply:
Yep, I agree there should be some instructions on the required Node version. Created #15 Issue to resolve this.